tests/test_job_management.py

```diff
@@ -1955,6 +1953,80 @@ def x(arg):
     assert len(my_globals['scwl_dict']['scw_list']) > max_length
 
 
+def test_email_t1_t2(dispatcher_long_living_fixture, dispatcher_local_mail_server):
+    from cdci_data_analysis.plugins.dummy_instrument.data_server_dispatcher import DataServerQuery
+    DataServerQuery.set_status('submitted')
+
+    server = dispatcher_long_living_fixture
+
+    DispatcherJobState.remove_scratch_folders()
+
+    token_payload = {
+        **default_token_payload,
+        "tem": 0
+            }
+    encoded_token = jwt.encode(token_payload, secret_key, algorithm='HS256')
+
+    dict_param = dict(
+        query_status="new",
+        query_type="Real",
+        instrument="empty-async",
+        product_type="dummy",
+        T1="57818.560277777775",
```

Comment:
What will happen with invalid format? 
Also what if `T1=57818.560277777775` and  `T2="2011-11-11T11:11:11"`?

Reply:
It really depends by what is specified (if it is!) inside `T_format`.
>
>  Also what if `T1=57818.560277777775` and `T2="2011-11-11T11:11:11"`?

it will throw an excepotion when it will attempt a conversion, depending on the value of `T_format`

Reply:
> It really depends by what is specified (if it is!) inside `T_format`.
> 
> > Also what if `T1=57818.560277777775` and `T2="2011-11-11T11:11:11"`?
> 
> it will throw an exception when it will attempt a conversion, depending on the value of `T_format`

I mean we'd need to test that. I think before format was detected automatically. There are many options.

Reply:
Yes of course, I will include some tests